readthedocs.yml

```diff
@@ -8,6 +8,10 @@ build:
       # we need the tags for versioneer to work
       - git fetch origin --depth 150
       - git fetch --tags
+    pre_build:
+      - PYDEVD_DISABLE_FILE_VALIDATION=1
+    post_build:
+      - mamba list
```

Comment:
this doesn't need to be here, but it's not exactly harmful, and might come in handy in future